examples/directml/stable_diffusion/stable_diffusion.py

```diff
@@ -188,13 +193,30 @@ def optimize(
 
     model_info = dict()
 
-    for submodel_name in ("text_encoder", "vae_encoder", "vae_decoder", "safety_checker", "unet"):
+    models_without_safety_checker = [
+        "stabilityai/stable-diffusion-2",
```

Comment:
You should be able to programmatically check if the model has a checker and avoid baking model names using something like this:

```python
pipeline.safety_checker is None
```